### ISSUE TITLE
License

### ISSUE THREAD

**Issue by theGiallo** [prime31/TouchKit#71]
I'm no expert, but I suspect the current license makes it unusable for companies that make experts check their licenses. It would be nice to hear an expert. Maybe picking a license for "use in games" and prohibiting the "use in similar libraries" would be good?

Probably picking MIT for everything would be better.

**Comment by svermeulen** [comment]
Yeah, the custom license as written seems very permissive (which is appreciated) but not using a standard license like MIT will make it unusable for many people
